### PR TITLE
Desktop version pull request

### PR DESCRIPTION
**Pull request by tufoinnkuo10**
In this project
- New desktop templates are used
-  New images used
-  Responsive transmission from the mobile to desktop version 
-  Read me edited as well

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -559,3 +566,241 @@ li a {
...
+.big-form-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.icons3 {
+  margin-top: 100px;
+}
+
+@media screen and (min-width: 991px) {
```
Comment (AshakaE) [inline style.css]: Media query to low

- [ ] Kindly select a value from `992 - 1100px` to target desktop screens

File: style.css [hunk 2]
```diff
@@ -559,3 +566,241 @@ li a {
...
+    height: 150px;
+    width: 500px;
+  }
+
+  .icons3 {
+    margin-top: 100px;
+  }
+}
+
+@media screen and (max-width: 991px) {
```
Comment (AshakaE) [inline style.css:770]: Media query to low

- [ ] Kindly select a value from `992 - 1100px` to target desktop screens

File: index.html [hunk 3]
```diff
@@ -1,312 +1,351 @@
...
-    <header>
-      <nav>
-        <ul class="nav">
-          <li><a href="/">TUFOIN</a></li>
+  <link rel="stylesheet" href="style.css" />
+  <title>Document</title>
+</head>
+
+<body>
+  <header>
```
Comment (AshakaE) [inline index.html:29]: - [ ] Kindly add more margins or padding to catch breaking changes. see image below
image

File: index.html [hunk 4]
```diff
@@ -1,312 +1,351 @@
...
+              <li class="gt btnn">Gitlab</li>
+              <li class="tr btnn">Terminal</li>
+            </ul>
+          </div>
         </div>
-      </section>
-      <section class="form-section">
+      </div>
+    </section>
+    <section class="form-section">
```
Comment (AshakaE) [inline index.html:266]: - [ ] Kindly add more margins or padding to catch breaking changes. see image below
image

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @tufoinnkuo10  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!

#  Required Changes ♻️ 🚧
_Check the comments under the review._

- [ ] Kindly fix the page overflow in mobile versions. see image below
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -1,312 +1,351 @@
...
           </li>
         </ul>
-      </nav>
-    </header>
+      </div>
+    </section>
+    <div class="work-section">
+      <h1 class="work-h1">My Recent Works</h1>
+      <div class="line"></div>
+    </div>
```
Comment (barackm) [inline index.html:84]: - [ ] [OPTIONAL] It seems that you have some issues with this title, see screenshots below 👇 
- Your design
image

- Figma design
image

Can you please fix this issue?

File: README.md [hunk 6]
```diff
@@ -1,15 +1,20 @@
...
+A Demo web page of  my self (Mobile/Desktop version)
 
-> My first project
-Designed a personal webpage about myself with a template from Figma.
+> Figma clone project
+Designed a personal Mobile and Desktop responsive webpage about myself with a template from Figma.
 
-![screenshot](portfolio.png)
+![screenshot](desktop.png)
+![screenshot](mobile.png)
```
Comment (barackm) [inline README.md:8]: - [ ] [OPTIONAL] it seems that these images are broken in the README, can you please fix this? 👍 
- [ ] [OPTIONAL] You can also update your demo link in to reflect the actual changes 😉

File: index.html [hunk 7]
```diff
@@ -1,312 +1,351 @@
...
+          <a href="/"><i class="fab fa-twitter"></i></a>
+        </li>
+        <li>
+          <a href="/"><i class="fab fa-medium-m"></i></a>
+        </li>
+      </ul>
+      <img class="line22" src="./image/line.png" alt="short-line">
+    </div>
+  </div>
+</body>
```
Comment (barackm) [inline index.html:349]: - [ ] [OPTIONAL] It seems that you have a big empty space after your footer, let's fix this please 👍 
image

**Review by barackm — APPROVED** [review]
Hi @tufoinnkuo10, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] Please let us give descriptive names to our branches, this will help people understand what are the expected changes for the branch.
- [ ] [OPTIONAL] Let's be consistent with our commit messages, I can see that you have some commit messages all in uppercase, some of them start with lowercase letters, Please let's always start with Uppercase letters for commit messages, and remember the commit message should not have all letters in uppercase. let's make better mistakes tomorrow 😉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**